README.md

```diff
@@ -1,2 +1,14 @@
 # sd-serverdiagnostics
 Server Connector For MageDiagnostics
+
+Installation Process:
+
+- Copy the repo contents to the /opt folder on each webserver
+
+- cp .env.sample .env
+
+- In the .env file, set the Server key from the server key value found for that client in the magediagnostics dashboard
+
+- In the .env file, set the `SD_MAGEDIAGNOSTICS_API_ENDPOINT` to `https://magediagnostics.com:1234/api/v1/server`
+
+- Add a cron tab entry `0 0 * * * sh main.sh`
```

Comment:
should be...

```
0 0 * * * /opt/sd-serverdiagnostics/main.sh
```

Need the correct path. You don't need `sh` (the shebang specifies how it should be executed).